src/main/java/com/jfrog/ide/idea/scan/data/applications/JFrogApplicationsConfig.java

```diff
@@ -22,15 +23,16 @@ public class JFrogApplicationsConfig {
 
     public static JFrogApplicationsConfig createApplicationConfigWithDefaultModule(Project project) {
         JFrogApplicationsConfig applicationsConfig = new JFrogApplicationsConfig();
-
-        ModuleConfig defualtModuleConfig = new ModuleConfig();
-        defualtModuleConfig.setSourceRoot(getProjectBasePath(project).toAbsolutePath().toString());
-        defualtModuleConfig.setExcludePatterns(convertToSkippedFolders(GlobalSettings.getInstance().getServerConfig().getExcludedPaths()));
-
+        Set<Path> paths = com.jfrog.ide.idea.scan.ScanUtils.createScanPaths(project);
```

Comment:
Better with regular import
```suggestion
        Set<Path> paths = ScanUtils.createScanPaths(project);
```